### PR TITLE
Fixed Typos

### PR DESCRIPTION
**Pull request by ergenekonyigit**

### CODE REVIEW

**Comment by udp** [comment]
Why did you remove all the double spaces after periods?  That's stylistic, not a typo.  It's particularly useful for readability in fixed width text.

**Comment by udp** [comment]
You also managed to remove some spaces _completely_.  I suggest proof reading next time.

File: docs/CallingConvention.rst [hunk 1]
```diff
@@ -12,10 +12,10 @@ want it to be.
 
 It's a basic assumption in this paper that Swift shouldn't make an
 implicit promise to exactly match the default platform calling
-convention.  That is, if a C or ObjC programmer manages to derive the
+convention. That is, if a C or Objective C programmer manages to derive the
```
Comment (krememek) [inline docs/CallingConvention.rst]: If we are going to remove the abbreviation, we should use the standard spelling which is "Objective-C".

File: docs/CallingConvention.rst [hunk 2]
```diff
@@ -1202,25 +1202,25 @@ Parameters
...
-parameters.  The latter can arise even in C and Objective-C.
+parameters. The latter can arise even in C and Objective-C.
 
 Otherwise, the rules are somewhat different if we're matching the C
-calling convention.  If the parameter is a nominal type imported from
+calling convention. If the parameter is a nominal type imported from
 Clang, then we just add the imported Clang type to the Clang function
-type as a parameter.  Otherwise, we derive the legal type sequence for
-the parameter type.  Again, we should only have compatibility
+type as a parameter.Otherwise, we derive the legal type sequence for
```
Comment (haubc) [inline docs/CallingConvention.rst]: Introducing a typo here with no space between two sentences.

File: docs/CallingConvention.rst [hunk 3]
```diff
@@ -1202,25 +1202,25 @@ Parameters
...
+the parameter type. Again, we should only have compatibility
 requirements if the legal type sequence has a single element, but for
 the convenience of interoperation, we collect the corresponding Clang
 types for all of the elements of the sequence.
 
 Finally, if we're matching the Swift calling convention, derive the
-legal type sequence.  If the result appears to be a reasonably small
+legal type sequence.If the result appears to be a reasonably small
```
Comment (haubc) [inline docs/CallingConvention.rst]: Introducing a typo here with no space between two sentences.

**Pull request closed without merging**